src/lib.rs

```diff
@@ -30,8 +30,6 @@ use js_sys::Promise;
 use spmc::{channel, Receiver, Sender};
 use wasm_bindgen::prelude::*;
 use wasm_bindgen::JsValue;
-
-#[cfg(feature = "no-bundler")]
 use js_sys::JsString;
```

Comment:
I don't think removal of this attribute was necessary, because you're still not using `JsString`?

Reply:
sorry, fixed